Description: Fix 00416_progress_in_http_headers test when using system libpoco
 Clickhouse send progress headers through custom embedded libpoco
 .
 Test fails when using libpoco from system because progress headers
 are not sent
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: other
Last-Update: 2017-03-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: clickhouse/dbms/tests/queries/0_stateless/00416_progress_in_http_headers.reference
===================================================================
--- clickhouse.orig/dbms/tests/queries/0_stateless/00416_progress_in_http_headers.reference	2018-01-21 16:15:24.049615529 +0100
+++ clickhouse/dbms/tests/queries/0_stateless/00416_progress_in_http_headers.reference	2018-01-21 16:15:24.045615522 +0100
@@ -1,14 +1,3 @@
-< X-ClickHouse-Progress: {"read_rows":"0","read_bytes":"0","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"1","read_bytes":"8","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"2","read_bytes":"16","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"3","read_bytes":"24","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"4","read_bytes":"32","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"5","read_bytes":"40","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"6","read_bytes":"48","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"7","read_bytes":"56","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"8","read_bytes":"64","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"9","read_bytes":"72","total_rows":"0"}
-< X-ClickHouse-Progress: {"read_rows":"10","read_bytes":"80","total_rows":"0"}
 0
 1
 2
